lib/Bio/EnsEMBL/DataCheck/Checks/CheckGenomicAlignTreeTable.pm

```diff
@@ -105,6 +105,30 @@ sub tests {
 
   }
 
+  my $sql_4 = qq/
+    SELECT COUNT(*)
+      FROM genomic_align_tree gat
+        JOIN genomic_align ga USING (node_id)
+        JOIN dnafrag df USING (dnafrag_id)
+      WHERE gat.right_index-gat.left_index!=1
```

Comment:
```suggestion
      WHERE gat.right_index - gat.left_index != 1
```
Formatting comments could potentially be ignored, but I find the additional spaces easier to read here in the raw sql